noisemodelling-emission/src/main/java/org/noise_planet/noisemodelling/emission/LineSource.java

```diff
@@ -62,12 +62,12 @@ private void setLW(double[] lW) {
     }
 
 
-    public LineSource(double[] lW, double sourceHeight, String typeSource, String directivity) {
+    /*public LineSource(double[] lW, double sourceHeight, String typeSource, String directivity) {
```

Comment:
If this function is never used and you want to remove it, delete the code not comment it out.